client/flutter/screenshots.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# INSTALLATION:
+# Install imagemagick (dependency for screenshots)
+# Install and active screenshots from master (fixes some issues with iOS simulators)
+#
+# STEPS:
+# brew update && brew install imagemagick
+# brew tap dart-lang/dart
+# brew install dart
+# pub global activate -s git https://github.com/mmcc007/screenshots.git
+#
+# RUNNING:
+# Run `screenshots`
+# Run `xcrun simctl shutdown all && screenshots` to close all simulators first
```

Comment:
Not a blocker, but would be helpful: can you quickly copy these steps into a documentation markdown file in the `docs/devdesign` folder?